### ISSUE TITLE
Azure DICOM Server Template Won't deploy to CanadaCentral

### ISSUE THREAD

**Issue by mwmah** [microsoft/dicom-server#1231]
Hi,

I took a stab at deploying the DICOM server using the Azure Portal as per
https://github.com/microsoft/dicom-server/blob/main/docs/quickstarts/deploy-via-azure.md

But I got an error
_Deployment template validation failed: ‘The provided value 'canadacentral' for the template parameter 'applicationlnsightsLocation' at line '84' and column ‘30’ is not valid. The parameter value is not part of the allowed value(s): 'southeastasia,northeurope,westeurope,eastus,southcentralus,westus2',', (Code: InvalidTemplate)_

Can the template paramter be adjusted to include 'canadacentral' please?

**Comment by jovinson-ms** [comment]
Hi @mwmah, thanks for trying out the DICOM server! You should be able to deploy the quickstart template to `canadacentral` by simply leaving the `applicationlnsightsLocation` template parameter empty.

While most of the resources can be deployed in any region, the template limits Application Insights deployments to a few regions, but will [select a region that is close to your deployment](https://github.com/microsoft/dicom-server/blob/746e6e9e099b3436d1609219259dbe6ca9a2510f/samples/templates/default-azuredeploy.json#L93).

I'll close this issue for now, but feel free to re-open if you're still having issues and I'll be happy to help unblock you.
